docs/src/modules/components/AppFrame.js

```diff
@@ -98,6 +99,16 @@ const styles = theme => ({
     marginLeft: theme.spacing(2),
     flex: '0 1 auto',
   },
+  skipNav: {
+    position: 'absolute',
+    transition: theme.transitions.create('top'),
```

Comment:
I would avoid the animation, but reducing the animation duration could be a good compromise, also using the `easeOut` easing curve is more appropriate.

Reply:
> I would avoid the animation

"Probably the most accessible method for visually hiding skip links is to hide them off screen, then cause them to be positioned on screen when they receive keyboard focus."

https://webaim.org/techniques/skipnav/#invisible

Animation attracts attention.

>  reducing the animation duration could be a good compromise, also using the easeOut easing curve is more appropriate.

You're right. It should be `theme.transitions.duration.enteringScreen`, and `easeOut`.

(Is theme.transitions documented anywhere? I had to check the source. It could be another good one for the css-utils package.)

Reply:
I agree, we should be documenting all our theme helpers 👍 . I don't know anymore what we should do with `css-utils`.